src/Views/MultiStepForm.jsx

```diff
@@ -402,7 +402,7 @@ function MultiStepForm(props) {
             {/* Load Touchpoints feedback form */}
             {step === 6 && lang === 'en' &&
                 <>
-                    <div id="touchpoints-form-embed" className={'margin-top-6'}></div>
+                    <div id="touchpoints-form-embed" className={'vote-touchpoints-margins'}></div>
```

Comment:
I think we can remove this class and target the module class instead

Reply:
I am just removing all together since the top margin will be set in the fba-inline-container class override